plutus-metatheory/src/Type/Equality.lagda.md

```diff
@@ -33,6 +33,13 @@ open import Type.RenamingSubstitution
 
 This serves as a declaritive specification of the semantics of types.
 
+We define type equality as an intrinsically scoped and kinded relation. In particular,
+this means it is impossible to state an equation between types in different contexts, or of
+different kinds. The only interesting rule is the β-rule from the lambda calculus. We omit
+the η-rule as Plutus Core does not have it. The additional
+types (`⇒`, `∀`, and `µ`) do not have any computational behaviour, and are essentially inert.
```

Comment:
Wait, I'm confused, is it `Π` or `∀`?

Reply:
We're using the symbol Pi, so this should say Pi too